modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/java/com/liferay/portal/workflow/metrics/internal/search/index/TaskWorkflowMetricsIndexerImpl.java

```diff
@@ -388,7 +391,10 @@ private void _deleteTask(long companyId, long taskId) {
 				).language(
 					"painless"
 				).putParameter(
-					"taskId", taskId
+					"task",
```

Comment:
Hey @MarinhoFeliphe, it's ok just pass the id when in a delete operation. So, please keep it in the old way. Thx